src/objects.rs

```diff
@@ -32,6 +32,9 @@ pub struct ObjectData {
     /// The name of the object, which is arbitrary and set by the user.
     pub name: String,
     /// The type of the object, which is arbitrary and set by the user.
+    pub user_type: String,
+    /// This property has been renamed to `user_type`.
+    #[deprecated(since = "0.11.0", note = "Use [`ObjectData::user_type`] instead")]
```

Comment:
Actually `current` refers to the next 0.10 version, so that would be 0.10.3.

Reply:
Oh, yeah, good catch! I have custom-patched 0.11 in my project and missed that 'current' is at 0.10